README.rst

```diff
@@ -54,18 +54,66 @@ Installation
 Usage
 -----
 
-Unit test report
-================
-
 .. code-block:: console
 
-    mlx.xunit2rst -i utest_report.xml -o utest_report.rst
+    mlx.xunit2rst -i itest_report.xml -o my_lib_report.rst --prefix ITEST_MY_LIB-
+
+    mlx.xunit2rst --help
+
+    usage: mlx.xunit2rst [-h] -i INPUT_FILE -o RST_OUTPUT_FILE [-s] [-p PREFIX]
+                         [--trim-suffix]
+                         [--unit-or-integration UNIT_OR_INTEGRATION] [-v]
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -i INPUT_FILE, --input INPUT_FILE
+                            The input XML file
+      -o RST_OUTPUT_FILE, --output RST_OUTPUT_FILE
+                            The output RST file
+      -s, --itemize-suites  Flag to itemize testsuite elements instead of testcase
+                            elements.
+      -p PREFIX, --prefix PREFIX
+                            Optional prefix to add to item IDs
+      --trim-suffix         If the suffix of the --prefix argument ends with '_-'
+                            it gets trimmed to '-'
+      --unit-or-integration UNIT_OR_INTEGRATION
+                            Optional: give value starting with 'u' or 'i' if the
+                            the script's discernment is wrong.
+      -v, --version         show program's version number and exit
 
-Integration test report
-=======================
+.. _`mlx.traceability`: https://pypi.org/project/mlx.traceability/
 
-.. code-block:: console
+--------
+Behavior
+--------
 
-    mlx.xunit2rst -i itest_report.xml -o lib_lin_itest_report.rst --prefix ITEST_LIB_LIN-
+Itemization
+===========
 
-.. _`mlx.traceability`: https://pypi.org/project/mlx.traceability/
+By default, all *testcase* elements from the input file are used to created treaceability items. This may not always be
+desired. The ``-s, --itemize-suites`` flag lets the script itemize *testsuite* elements instead. In this case, the
+*testcase* elements will still be parsed to determine whether the testsuite passed or failed.
+
+Item names
+==========
+
+The *name* attribute of the element to be itemized is used to build the item ID. Lower case letters get converted to
+upper case, whitespaces get converted to underscores, and *&* characters get converted to *AND*. A valid prefix must
+end with a hyphen to be recognized by the script.
+
+Prefix
+======
+
+Traceability item IDs have a prefix that is unique for the group they belong to, e.g. *ITEST_MY_LIB-*. The``--prefix``
+input argument lets you configure this prefix. It will be prepended to the item names found in the input file to build
+the item ID. By default, the script adds _ITEST-_ or _UTEST-_ for integration or unit test reports, unless the prefixes
```

Comment:
Can you emphasize that it adds underscore? Maybe:

```
the script adds _ITEST-_ or _UTEST-_ (note underscores)
```

Reply:
Oops. I meant to use italic type here so I should've added * like so `*ITEST-*`
Fixed in e6b30a472076b51ae38573387da441ab5c069690